include/fileurlreader.h

```diff
@@ -11,13 +11,13 @@ class FileUrlReader : public UrlReader {
 public:
 	explicit FileUrlReader(const std::string& file = "");
 
-	void reload() override;
+	nonstd::optional<std::string> reload() override;
 	std::string get_source() override;
 
 	/// \brief Write URLs back to the input file.
 	///
 	/// This method is used after importing feeds from OPML.
-	void write_config();
```

Comment:
It'd be nice to update the docs and explain what this function returns. Ditto with other updated functions. (It's up to you if you'd like to write docs for the functions that you changed which didn't have a doc yet.)

Reply:
Does https://github.com/newsboat/newsboat/pull/1245/commits/d09f9e3fd3ca978e0a20e63bdeb906c9592baf82 suffice?

Reply:
Yep, looks good, thank you!